src/components/CaseStudies/StaffSpotlight.tsx

```diff
@@ -35,23 +35,22 @@ export const AuthorBio: React.FunctionComponent<{
                 </div>
             </div>
         ) : (
-            // Single author design
+            // Single staff design
             <div className="tw-flex tw-flex-col md:tw-flex-row">
-                {authors[0].image && (
+                {staff[0].image && (
                     <div className="col-md-3">
```

Comment:
I think we need another treatment here since the avatar should be left aligned without padding on the outer lef/right sides. There also needs to be a `sm` margin of spacing between the avatar and the staff name.

![Screen Shot 2022-09-01 at 10 22 31 AM](https://user-images.githubusercontent.com/1733936/187984828-39b1510e-1a2c-4929-b0af-e5487c6152f8.png)

Reply:
Good catch! Did you just eyeball to come up with the `sm` vertical spacing? I'm not seeing the space indicated here on Figma on the spacing page or this case study page. Is our standard `sm` spacing between differing items in a section unless otherwise specified? @bretthayes

Reply:
I inspected the Figma while looking at the mobile version and noticed it. If you hover your mouse between elements, you can see the spacing used and then I just reference our spacing in the DLS.

![Screen Shot 2022-09-01 at 3 46 08 PM](https://user-images.githubusercontent.com/1733936/187999540-bd4e4b62-fec6-4e8e-a304-12a4340491d5.png)

Also, a cool tidbit; if you type `ctrl + space` before typing a class name, the intellisense shows you the values in VSCode and you can keyboard through them 🤓 

![Screen Shot 2022-09-01 at 3 48 58 PM](https://user-images.githubusercontent.com/1733936/187999983-ac43e521-4bc5-418c-a866-b2fe45f5f9ae.png)